drivers/lte_link_control/Kconfig

```diff
@@ -83,6 +83,19 @@ config LTE_PDP_CONTEXT
 		The +CGDCONT command defines Packet Data Protocol (PDP) Context.
 		For reference, see 3GPP 27.007 Ch. 10.1.1
 
+choice
+	prompt "Select network mode"
```

Comment:
Should this be system mode instead of network mode? (Based on the AT command.)

Reply:
The AT command used ("XSYSTEMMODE") is used to enable/disable LTE-M, NB-IoT and GPS modes. In this module we are only dealing with LTE-M and NB-IoT, so "network mode" is perhaps more  descriptive in this particular context?

Reply:
OK - will change this in the documentation as well then.